sqrl-tools/sqrl-cli/src/main/java/com/datasqrl/cmd/DiscoverCommand.java

```diff
@@ -67,8 +76,11 @@ protected void runCommand(ErrorCollector errors) throws IOException {
       statistics = false;
     }
 
-    SqrlConfig config = PackagerUtil.getOrCreateDefaultConfiguration(root, errors,
-                ()-> PackagerUtil.createEmbeddedConfig(root.rootDir, errors));
+    //check package json, or use embedded config
+    SqrlConfig config = Packager.findPackageFile(root.rootDir, this.root.packageFiles)
+        .map(p -> SqrlConfigCommons.fromFiles(errors, p))
+        .orElseGet(() -> createEmbeddedConfig(errors));
```

Comment:
@mbroecheler Should discovery resolve profiles? I see arguments for both ways.

Reply:
I think of discover as something that folks run locally to get their data packages without much work. Hence, I don't think we need the full flexibility of profiles. At least not yet.